.dep-versions

```diff
@@ -8,7 +8,7 @@ enzyme=v0.0.149
 
 # For a custom PL version, update the package version here and at
 # 'doc/requirements.txt
-pennylane=0.42.0-dev19
+pennylane=0.42.0-dev24
 
 # For a custom LQ/LK version, update the package version here and at
 # 'doc/requirements.txt'
```

Comment:
Make sure to update the lightning dependency as well. It should now be `0.42.0-dev16`